### ISSUE TITLE
Screen reader is not giving information about the number of cards in a column.

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1201]
**Describe the bug**
Screen reader is not giving information about the number of cards in a column. The user is unable to determine the number of cards in a column, hindering their ability to assess the content or perform tasks efficiently. This impacts overall accessibility and usability for screen reader users.

**Are you using the Retrospectives Tool on-prem?**
No

**To Reproduce**
1. open the URL https://dev.azure.com/eDADPlayground/SCRUMProject .
2. Expand left navigation and select boards-->Retrospectives.
3. Navigate with Screen reader on the groups.
4. Verify whether screen reader is giving information about the number of cards in a column or not.

**Expected behavior**
Screen reader should give information about the number of cards in a column.

**Screenshots**

https://github.com/user-attachments/assets/1bf919c9-d94a-4103-a101-846f1eba07b1

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/ab0becf6-183b-4dc7-815a-feb3617c6c16" />

**What screen type are you seeing this on ?**
 Desktop Windows 11

**Additional context**
